### DISCUSSION TITLE
Vercel Preview Deployments vs Projects

### DISCUSSION THREAD

**Discussion by lohrm-stabl** [vercel/community#6482] [Help]
### Summary

Hello,

we have one project but we use three different branches to deploy different versions of it:
| Environment | Domain            | Git Branch |
|-------------|-------------------|------------|
| Testing     | test.ourdomain.com| `test`     |
| Staging     | stage.ourdomain.com| `stage`   |
| Production  | prod.ourdomain.com| `prod`     |

I wonder what the best strategy to deploy that onto Vercel would be.
Would one environment be enough, and we ignore the production / development deployments, but use custom domains to route to the test, stage and prod branches as preview deployments?
Are there any downsides to this? Do preview deployments not run in "production" mode?

Or should we create three different projects, each using its own branch as production deployment?

### Steps to Reproduce

nothing to reproduce

**Comment by versecafe** [comment]
@lohrm-stabl If you are tying the prefix directly to a given git branch then you should be fine to just do this all in one project and per branch set prod.domain.com on branch main, stage.domain.com on branch staging, etc etc
<img width="1380" alt="image" src="https://github.com/vercel/community/assets/147033096/e2b3c763-931a-4b7a-abca-b30ac711f369">

**Reply by lohrm-stabl** [reply]
Yeah, but then they are still called "preview deployments":
![image](https://github.com/vercel/community/assets/111898561/30931735-19fb-45ec-a0bd-c355d0a19c7e)

Would be good to know if this has any influence on caching, CDN etc.
Also seems like some features like deployment protection distinguish between production and preview environments.
